src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const personPartnerText = isMarried
+    ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```

Add indents so your code will become easier to read.